perun-base/src/main/resources/perun-roles.yml

```diff
@@ -4772,24 +4856,35 @@ perun_policies:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Facility
 
   addDestinationsDefinedByHostsOnFacility_Service_Facility_policy:
     policy_roles:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Facility
+      - MFA: Host
```

Comment:
we're not affecting the Host object itself, just reading from it, so no need for this check here. same for the methods below